templates/default/event/eventmodify.html

```diff
@@ -113,11 +113,15 @@ <H1>{t a=$event.ticketid}Add new event to request #$a{/t}</H1>
 					{/box_row}
 
 					{box_row icon="img/users.gif" label="Users:" field_class="multiselect"}
-						<SELECT name="event[userlist][]" class="lms-ui-multiselect" multiple data-default-value="{trans("- select user -")}" data-shorten-to-default-value="false">
+                                               <SELECT name="event[userlist][]" {tip class="lms-ui-advanced-select"} multiple data-shorten-to-default-value="false">
+                                                {foreach $userlist as $user}
+                                                {if !ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') || (ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') && $user.access) || in_array($user.id, $filter.owner)}
+                                                <OPTION{if !$user.access} disabled{/if} value="{$user.id}"{if is_array($event.userlist) && in_array($user.id, $event.userlist)}} selected{/if}>{$user.rname|truncate:40:"...":true} ({$user.login})</OPTION>
+                                                {/if}
+                                                {/foreach}
+
 							{foreach $userlist as $user}
```

Comment:
A dla rtticketmodify jest:
https://github.com/chilek/lms-plus/blob/ff4bc6dd526dc43b3e650c7fd7089718804ac7a1/templates/default/rt/rtticketmodify.html#L134-L140
Dlaczego tu jest inaczej?

Reply:
Czeka na akceptację: #1704 w ramach wydzielania małych zmian. 
Tam jest to zrobione identycznie jak w rtticketmodify. 
Jak wleci PR to zrobię merge i tutaj się to również zmieni.

Reply:
Github wprowadził fajną funkcję drafts. 
Proszę więc o nie robienie review dla draftów, od teraz będę konwertował PR, które czekają na coś. Ja drafty sprawdzam sobie travisem.